nevergrad/benchmark/experiments.py

```diff
@@ -586,6 +644,18 @@ def yabbob(seed: tp.Optional[int] = None, parallel: bool = False, big: bool = Fa
                     yield xp
 
 
+@registry.register
+def yaconstrainedbbob(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of yabbob with higher dimensions."""
+    for i in range(8):
+        internal_generator = yabbob(seed, constraints=i)
+        j = 0
+        for xp in internal_generator:
+            j = j + 1
+            if j % 8 == i:  # We use only one test case out of 4, due to computational cost.
```

Comment:
I'm not sure I would hardcode 8 everywhere, don't you plan on adding more cases?

Reply:
No plan for now, but using len(...) is simple, let me do this.

Reply:
+the constant 4 does not have to be equal to len(...)/2, that's just an arbitrary choice.